src/frontends/tensorflow_common/src/op/unique_with_counts.cpp

```diff
@@ -0,0 +1,37 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "common_op_table.hpp"
+#include "openvino/op/unique.hpp"
+
+using namespace std;
+using namespace ov::op;
+
+namespace ov {
+namespace frontend {
+namespace tensorflow {
+namespace op {
+
+
+OutputVector translate_unique_with_counts_op(const NodeContext& node) {
+    default_op_checks(node, 1, {"UniqueWithCounts"});
+
+    // get input 'x' from node and node name
+    auto x = node.get_input(0);
+    auto node_name = node.get_name();
+
+    auto unique = make_shared<v10::Unique>(x, false, ov::element::i32, ov::element::i32);
```

Comment:
Please get the name from `node` and set it for this new operation. Here is example from another translator: https://github.com/openvinotoolkit/openvino/blob/e2e9d6056f066e77722265e17c81a7071e0804c7/src/frontends/tensorflow_common/src/op/fft.cpp#L53

Reply:
@popovaan, thanks for the feedback! Just pushed requested changes.